src/backend/Shared/EventBus/EventBus.Abstraction/InMemoryEventBusSubscriptionsManager.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+
+using EventBus.Abstraction.Events;
+using EventBus.Abstraction.Models;
+using EventBus.Abstraction.Interfaces;
+
+namespace EventBus.Abstraction
+{
+    public partial class InMemoryEventBusSubscriptionsManager : IEventBusSubscriptionsManager
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4794247)